kiali/kiali
# ztunnel logs with multiple nodes

### Describe the change

Create an array of pod names from the same namespace from streamParsedLogs to be able to stream the logs from different nodes for ztunnel. 

Include the ztunnel pod in the log line: 

![image](https://github.com/kiali/kiali/assets/49480155/b7bb69ca-7d8e-40f2-907e-865960ce1048)
 

### Steps to test the PR

Create a Ambient kind cluster with more than one node. Then:

1.     istio/download-istio.sh -div 1.22-dev
2.     Install Istio: istio/install-istio-via-istioctl.sh -c kubectl -cp ambient -it default -ih default
3.     Install Kiali
4.     Install bookinfo: istio/install-bookinfo-demo.sh -c kubectl -ai false -tg

Go to the logs page, click in ztunnel logs. Add different combinations of number of lines, change the containers/log options.
ztunnel logs should be visible.

### Automation testing

existing integration tests. 2e2 test will be added for Istio 1.22.

### Issue reference

Fixes https://github.com/kiali/kiali/issues/7338
